packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -237,7 +241,6 @@
     position: absolute;
     bottom: 20px;
     width: calc(100% - #{$mdc-text-field-icon-padding});
-    color: $mdc-text-field-box-secondary-text;
```

Comment:
Why does this line need to be removed? It doesn't appear to be related to the `:focused` state.

Also, it looks like `$mdc-text-field-box-secondary-text` isn't used anywhere else, so if we really do need to remove this line, we should also delete the variable from `_variables.scss`.

Reply:
The default `color` property is set in the mdc-text-field-label.scss file. My idea was to let the label scss file determine the default values, and then override them when we need to change them based on interaction with the other elements in the mdc-text-field.scss file.